src/components/Select/Select.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+import Icon from "../Icons/Icon";
+
+export interface SelectProps {
```

Comment:
should this also take a `className` prop? I sort of feel that almost all components should